src/ClientInterface.ts

```diff
@@ -3,6 +3,8 @@ import { IJSONRPCNotification } from "./Request.js";
 interface Arguments {
   readonly method: string;
   readonly params?: readonly unknown[] | object;
+  readonly timeout?: number;
```

Comment:
I think we'd keep this and signal out of Arguments, to keep all the JSON-RPC specific arguments aligned together.

Reply:
Yes that's what we're discussing here https://github.com/open-rpc/client-js/pull/368#discussion_r2624878393